File: js/lista1/Desafio7.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
}); 

/*
Desafio7- Faça um programa que receba o valor do salário mínimo:

O valor do salário de um funcionário;

Calcule e mostre a quantidade de salários mínimos que ganha esse funcionário.
*/

rl.question('O valor do salário do salário mínimo é: ', (salarioMinimo) => {
    rl.question('O valor do salário do funcionário é: ', (salarioFuncionario) => {
        //let quantidadeSalario = salarioFuncionario / salarioMinimo;
    
        console.log(`A quantidade de salários minímos do funcionário é: ${salarioFuncionario / salarioMinimo}`);
      rl.close();
    });
  });

File: c/Lista1/Desafio7.c
cmd#include <stdio.h>
#include <stdlib.h>
#include <math.h>

main(){

    float SalarioMinimo, SalarioF, QuantSalMin;
    
    printf("Digite o valor do salário minimo: ");
    scanf("%f", &SalarioMinimo);

    printf("Digite o valor do salário do funcionário: ");
    scanf("%f", &SalarioF);

    QuantSalMin = SalarioF / SalarioMinimo;
    printf("A quantidade de salário minimo do funcionário é: %2.0f", QuantSalMin);
    
}
